src/main/java/org/broadinstitute/hellbender/cmdline/CommandLineProgram.java

```diff
@@ -53,7 +55,8 @@
  *
  */
 public abstract class CommandLineProgram implements CommandLinePluginProvider {
-    protected final Logger logger = LogManager.getLogger(this.getClass());
+
+    protected final Logger logger = LogManager.getLogger(CommandLineProgram.class);
```

Comment:
I changed it because that was the convention that @droazen said we follow.  

Since it's an abstract class I could go either way.  @lbergelson What do you think?